src/test/java/uk/gov/companieshouse/api/accounts/utility/TestEnvironmentSetupHelper.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.companieshouse.api.accounts.utility;
+
+import java.lang.reflect.Field;
+import java.util.Map;
+
+public interface TestEnvironmentSetupHelper {
```

Comment:
Perhaps put a JavaDoc description somewhere in this class to detail that it can be used for testing to set environment variables?